generators/bundler-webpack/templates/webpack-prod.config.js

```diff
@@ -0,0 +1,54 @@
+const path = require('path');
+const webpack = require('webpack');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const OptimizeCssAssetsPlugin = require('optimize-css-assets-webpack-plugin');
+
+module.exports = {
+  resolve: {
+    extensions: ['*', '.js', '.jsx', '.json'],
+  },
+  entry: {
+    main: [
```

Comment:
What about allowing for multiple entries with a globbed prefix like I do in the Django generator [here](https://github.com/The-Politico/generator-politico-django/blob/master/generators/app/templates/webpack-dev.config.js#L10-L16).